config/checkstyle/suppressions.xml

```diff
@@ -4,5 +4,13 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
-  <!--add any suppressions here -->
+  <suppress checks=".*" files=".*Test.java"/>
+  <suppress checks=".*" files=".*[\\/]unit[\\/]"/>
+  <suppress checks="JavadocVariable" files="MicrosoftStsIdToken.java"/>
+  <suppress checks="EmptyBlock" files="AzureActiveDirectoryOAuth2Strategy.java"/>
+  <suppress checks="EqualsHashCodeCheck" files="Account.java"/>
```

Comment:
This should not have been merged without CR -- this is not the proper syntax to omit `equals()` and `hashCode()` from checkstyle

From the docs:
>Checks that classes that either override equals() or hashCode() also overrides the other. This checks only verifies that the method declarations match Object.equals(Object) and Object.hashCode() exactly to be considered an override. This check does not verify invalid method names, parameters other than Object, or anything else. <br /><br />Rationale: The contract of equals() and hashCode() requires that equal objects have the same hashCode. Hence, whenever you override equals() you must override hashCode() to ensure that your class can be used in collections that are hash based.